src/CalcManager/CEngine/scicomm.cpp

```diff
@@ -103,6 +103,14 @@ void CCalcEngine::ProcessCommandWorker(OpCode wParam)
         m_nTempCom = (int)wParam;
     }
 
+    // Clear expression shown after = sign, when user do any action.
+    static bool afterEQUCommand = false;
```

Comment:
It is preferable here to use an instance field (in `CalcEngine.h`) rather than a static.

Reply:
I suppose you can use `m_bNoPrevEqu` instead and write `if(!m_bNoPrevEqu)` (need to be tested)